File: Data Structures & Algorithms/two-integer-sum/submission-0.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        ArrayList<Integer> temp = new ArrayList<Integer>(); 
         
        for(int i = 0; i < nums.length; i++){
            for(int j = i+1; j < nums.length; j++){
                if(nums[i] + nums[j] == target){
                    temp.add(i); 
                    temp.add(j); 
                }
            }
        }
        int[] solution = new int[temp.size()]; 
        for(int i = 0; i < solution.length; i++)
            solution[i] = temp.get(i); 

        return solution; 
         
    }
}

File: Data Structures & Algorithms/binary-search/submission-0.py
class Solution:
    def search(self, nums: List[int], target: int) -> int:
        l,r = 0, len(nums)-1
        mid = (l+r) // 2

        if target not in nums:
            return -1

        while nums[mid] != target:
            if target > nums[mid]:
                l = mid+1
                mid = (l+r) // 2
            if target < nums[mid]:
                r = mid-1
                mid = (l+r) // 2
        
        return mid

